src/views/MyRequests/Table.tsx

```diff
@@ -219,7 +219,7 @@ const Table = () => {
 
   return (
     <div className="accessibility-requests-table">
-      <UswdsTable bordered={false} {...getTableProps()} fullWidth>
+      <UswdsTable bordered={false} {...getTableProps()} fullWidth scrollable>
```

Comment:
What was this change for? I tried to load up my DB with a bunch of requests and didn't see any scrolling behavior.

Reply:
If for mobile responsive behavior.  I confirmed this with UX.  On mobile if the table isn't set to scrollable, the Navigation gets pulled back.